src/blue.ts

```diff
@@ -60,10 +60,16 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
     if (isFunction(target)) {
         // this new shadow target function is never invoked just needed to anchor the realm
         try {
-            shadowTarget = 'prototype' in target ? function () {} : () => {};
+            if ('prototype' in target) {
+                shadowTarget = hasOwnProperty(target, 'arguments')
+                    ? function () {}
+                    : function () {'use strict';};
+            } else {
+                shadowTarget = () => {};
+            }
         } catch {
             // target is a revoked proxy
-            shadowTarget = () => {};
+            shadowTarget = function () {'use strict';};
```

Comment:
I switched to a strict mode function so that it at least has a prototype property